src/Mapping/Operator/Factory/InputQuantityValue.php

```diff
@@ -59,8 +65,10 @@ public function evaluateReturnType(string $inputType, int $index = null): string
      */
     public function generateResultPreview($inputData)
     {
-        if ($inputData instanceof \Pimcore\Model\DataObject\Data\QuantityValue) {
-            return 'InputQuantityValue: ' . $inputData->getValue() . ' ' . ($inputData->getUnit() ? $inputData->getUnit()->getAbbreviation() : '');
+        if ($inputData instanceof \Pimcore\Model\DataObject\Data\InputQuantityValue) {
+            return 'InputQuantityValue: ' .
+                $inputData->getValue() . ' ' .
+                ($inputData->getUnit() ? '['.$inputData->getUnit()->getAbbreviation().']' : '');
```

Comment:
only `unitI`d would be set when passing a `string` in constructor and here would only appear the value without the unit

https://github.com/pimcore/pimcore/blob/89d0af7753f242378c22aab19ec00f678be3d780/models/DataObject/Data/AbstractQuantityValue.php#L48